src/canvas.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, application):
                      self.top_right_axis]:
             axis.get_xaxis().set_visible(False)
             axis.get_yaxis().set_visible(False)
-        self.dummy_toolbar = DummyToolbar(self)
+        DummyToolbar(self)
```

Comment:
We still create the dummy toolbar, but never assign it. Is this so that we can still use the overwritten rubberband functions?